src/app/utils/increaseSize.ts

```diff
@@ -0,0 +1,4 @@
+// rem * 10px * 8/7, then convert back to rem
+const increaseSize = (size: number) => Math.round(size * 10 * 8 / 7) / 10;
```

Comment:
@TomWoodward I'm wondering if this should live somewhere else and if I should specify the ratio in the function name.

Reply:
what even is all this math for?